File: src/2011.final-value-of-variable-after-performing-operations.2137/solution.ts
function finalValueAfterOperations (operations: string[]): number {
    let result = 0;
    for (let i = 0; i < operations.length; i++) {
        switch (operations[i]) {
        case 'X++':
        case '++X':
            result++;
            break;
        case 'X--':
        case '--X':
            result--;
        }
    }

    return result;
}

File: src/2011.final-value-of-variable-after-performing-operations.2137/solution.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int result = 0;
        for (String s : operations) {
            if (s.charAt(0) == '+' || s.charAt(2) == '+') {
                result++;
            } else {
                result--;
            }
        }
        return result;
    }
}